src/actors/runtime/AbstractActor.ts

```diff
@@ -95,8 +95,8 @@ export default abstract class AbstractActor {
   async registerActorReminder<_Type>(
     reminderName: string,
     dueTime: Temporal.Duration,
-    period: Temporal.Duration,
-    ttl?: Temporal.Duration,
+    period: Temporal.Duration | undefined,
+    ttl?: Temporal.Duration | undefined,
```

Comment:
`ttl?` makes the type optional, so ` | undefined` is not required https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#optional-properties

We can also make `period` as an optional type.

[playground link](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAcwE4FN1QBRgIYC26AXIgM5SoxjKIA+i4AJusNekwJSIDeAUIkGIICMnAA26AHTi4ybAAMAEunGyANIgAkPfEQC+CzgG4++vn1CRYCFBiwB1GFAAWAeQAON-OIAKeVEJcQnQAflIKKhpufiFhUQlpWXllVQ1tXRDDEzMLNEwcACIAKTgXMEKc-KxsZlZ2LlM+aqgnV09vPD8AoJKyiqr7Vud3L3gff0CCWrAWNjAOQYK20c7uqewTIA)

Reply:
Thanks, you are right